CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.10)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake)
 
 # set the project name
-project(uhh VERSION 1.0)
+project(uhh VERSION 1.0 LANGUAGES CXX)
```

Comment:
even though `LANGUAGES` is optional for cmake [project statements](https://cmake.org/cmake/help/latest/command/project.html), my build was failing because it could not detect it.

For future reference: https://cliutils.gitlab.io/modern-cmake/